### ISSUE TITLE
Running Stella with the smallest resolution

### ISSUE THREAD

**Issue by mowses** [stella-emu/stella#811]
I am trying to run Stella in ubuntu with the smallest resolution possible, but It seems to be impossible.
I read that the resolution is 160x192 pixels, so I am trying to run Stella using this size.
When I run Stella, the screen goes to 640x480 even If I set config flags to -tia.zoom 100.

How can I force a screen size to fit my needs?

**Comment by thrust26** [comment]
Won't work. 200% zoom is the lower limit. Else the UI will break. 

Also the Atari pixels are not square, so 160x192 would look very odd.

**Comment by mowses** [comment]
Is not possible even editing the source code? I mean, I am not worry about the UI itself... I also tried to force window resizing programatically but with no success.
My idea behind this question is because I am trying to send video streaming to a server and 640x480 is a lot more data to send over network/internet for each frame than 160x192 would. In fact, It would stream "unnecessary pixels" to the clients.

**Comment by thrust26** [comment]
I suppose the data streamed is compressed. And then, since the information stays the same, the amount of data transferred should be about the same too. Especially if you disable a post processing (scaling, TV effects etc.). 

I haven't looked into the code, but I would expect a lot off assumptions failing if you reduce the minimal size. So there is quite a chance that this would break a lot of code.

**Comment by mowses** [comment]
@thrust26 its true that disable scalling, tv effects, etc.. would result in smaller sizes. Thus compressing the video and audio would result in the minimal size for streaming. The downside of that is the growing of the latency.
Some weeks ago I have edited the source files of TIA.hxx/cxx and used to broadcast via UDP the "pixel map" of each frame Stella produces. I also have used a technique to make the transfer of data even smaller using a custom "filter" for each pixel and using a indexed table for each pixel color. The final result was something like this:

```
[[00000,000],[33333,127],[44444,055]...]
```
What this means is:
- 00000, 33333 and 44444 are the pixel start;
- 000,127 and 055 are the indexed color from the TIA table - the palette has 128 colors;

I used a html `<canvas>` to render the video using this data and it works pretty well. In this example, the canvas renders from pixel 0 up to 33332 with the indexed color 0 - (lets say its red). then, again, from 33333 up to 44443 with indexed color 127.. and so on until the last pixel ... The reason I padded zeros at left is to represent fixed length UDP data.

This seems very weird but it really works. (works because of Stella has indexed table for colors, otherwise I would need to stream  it the "orthodoxal" way).

Well, now I just trying to stream Stella using FFMPEG just to see the difference: latency/packet size, etc... But seems clearly that the custom way would result in a better result.
